### PR TITLE
🔥 write integer Query Parser.

### PR DESCRIPTION
**Pull request by dozheiny**
This pull request adds a new feature to the codebase - an Integer Query URL Parser. This parser extracts all integers found in the query parameters of a given URL. The parser takes in a single argument, the URL, and returns a list of integers found in the query parameters. This feature can be useful for extracting and analyzing data from web analytics and other online tracking systems. The code has been tested and is ready for review and integration into the main branch.

### CODE REVIEW

File: ctx_test.go [hunk 1]
```diff
@@ -2119,6 +2119,10 @@ func Test_Ctx_Query(t *testing.T) {
 	utils.AssertEqual(t, "john", c.Query("search"))
 	utils.AssertEqual(t, "20", c.Query("age"))
 	utils.AssertEqual(t, "default", c.Query("unknown", "default"))
+	utils.AssertEqual(t, 0, c.QueryInt("20"))
```
Comment (ReneWerner87) [inline ctx_test.go:2122]: please more this to a new test -> don´t mix it

File: helpers.go [hunk 2]
```diff
@@ -143,6 +144,28 @@ func defaultString(value string, defaultValue []string) string {
 	return value
 }
 
+// defaultInt returns the value or a default value if it is set.
+func defaultInt(value string, defaultValue []int) int {
```
Comment (ReneWerner87) [inline helpers.go:148]: defaultInt should have an int as first value and the conversion should take place outside of it

File: helpers.go [hunk 3]
```diff
@@ -143,6 +144,28 @@ func defaultString(value string, defaultValue []string) string {
...
+	if len(value) == 0 {
```
Comment (ReneWerner87) [inline helpers.go:149]: maybe you can then use this method also at ParamsInt

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]
some little improvements todo

File: ctx_test.go (hunk 1 above)
Comment (dozheiny) [reply ctx_test.go:2122]: Ok Sure.

**Comment by dozheiny** [comment]
> some little improvements todo

Sure, I will commit to the changes you asked for.

**Comment by ReneWerner87** [comment]
thx

**Comment by dozheiny** [comment]
I commit changes you asked for.

File: ctx_test.go (hunk 1 above)
Comment (ReneWerner87) [reply ctx_test.go]: you forgot this

Comment (dozheiny) [reply ctx_test.go]: Done.

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**